.github/workflows/deploy.yml

```diff
@@ -26,9 +26,17 @@ on:
         description: "Has to be main or in format <project>-<ticketnumber>, e.g. spsh-130"
         default: main
         type: string
+      playwright_branch:
```

Comment:
Do we need this variable? We already have the branch in the other _branch inputs, I think it would be better to get the branch from there and have one less variable.

Reply:
@simoncolincap I removed the 'playwright_branch' input